code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -30,7 +30,7 @@
 	melee_damage_upper = 30
 	see_in_dark = 8
 	var/boost = 0
-	bloodcrawl = BLOODCRAWL_EAT
+	bloodcrawl = 2 //For some reason, the define doesn't work here.
```

Comment:
Yes that's why it doesn't work because #defines only work on code physically after the define.

That's why the __DEFINES folder isn't the DEFINES folder, because then it wouldn't be read first.